Clients/src/Swag4Net.Core/ApiModel.fs

```diff
@@ -0,0 +1,113 @@
+namespace Swag4Net.Core
+
+open System.Net
+
+module ApiModel =
+  type TypeName = string
+  type Any = string
+  type RegularExpression = string
+
+  type Api =
+    { Infos:Infos
+      Host:string
+      BasePath:string
+      Routes:Route list
+      Definitions:Schema list }
+  and Infos =
+    { Description:string
+      Version:string
+      Title:string
+      TermsOfService:string
+      Contact:Contact option
+      License:License option }
+  and Contact = 
+    | Email of string
+  and License = 
+    { Name:string; Url:string }
+  and Schema =
+      {
+        Type: DataType
+        Title: string option
+        AllOf: Schema option
+        OneOf: Schema option
+        AnyOf: Schema option
+        Not: Schema option
+        MultipleOf: Schema option
+        Items: ItemType option
+        Maximum: int option
+        ExclusiveMaximum: int option
+        Minimum: int option
+        ExclusiveMinimum: int option
+        MaxLength: int option
+        MinLength: int option
+        Pattern: RegularExpression option
+        MaxItems: int option
+        MinItems: int option
+        UniqueItems: bool option
+        MaxProperties: int option
+        MinProperties: int option
+        Properties: Map<string, Schema> option
```

Comment:
Please use `PropertyType` 
https://github.com/rflechner/Swag4Net-1/blob/master/Clients/src/ClientsForSwagger.Core/Models.fs#L32
In domain, property type are resolved and discriminated

Reply:
don't really understand what you mean, property are just tagged association betwwen two known types